src/hpnmg/modelchecker/RegionModelChecker.h

```diff
@@ -13,6 +13,7 @@
 #include "modelchecker/Negation.h"
 #include "modelchecker/Until.h"
 #include "ParametricLocationTree.h"
+#include "util/Logging.h"
```

Comment:
Better move this to the `cpp` file as this it is not part of the interface but only the implementation.